tools/memleak.py

```diff
@@ -206,6 +236,19 @@ def run_command_get_pid(command):
 bpf_source = open("memleak.c").read()
```

Comment:
... or consider inlining. There's a few .c files in /tools from the earlier days, but I'll do an update and merge them into the .py.